### PR TITLE
Update Managing_DCP_keys.md

### PR DESCRIPTION
**Pull request by KevinVerm**

### CODE REVIEW

**Comment by KevinVerm** [comment]
@baptistepa can also give your blessing?

File: user-guide/Advanced_Modules/Dashboards_app/Creating_and_configuring_dashboards/Sharing_a_dashboard.md [hunk 1]
```diff
@@ -26,11 +26,13 @@ uid: Sharing_a_dashboard
 
 1. If it is the first time you are sharing the dashboard, you may be asked to confirm that you want to link your account to the cloud. Select *I want to link the above users* and click *Link accounts*.
 
-1. In the pop-up window, fill in the email address of the person you want to share the dashboard with.
+1. In the pop-up window, *With* field, fill in the email address of the person you want to share the dashboard with.
```
Comment (baptistepa) [inline user-guide/Advanced_Modules/Dashboards_app/Creating_and_configuring_dashboards/Sharing_a_dashboard.md:29]: I would reword this as:
"In the pop-up window, fill in the email address of the person(s) you want to share the dashboard with in the *With* field.

**Review by baptistepa — COMMENTED** [review]
Reviewed, minor comment.

**Comment by MariekeGO** [comment]
@KevinVerm, could you check the reviewed version to see if everything is still OK?

**Comment by KevinVerm** [comment]
https://github.com/SkylineCommunications/dataminer-docs/pull/431/commits/d236a8d341149274cf646a20a4cc1468dbfecd18 looks ok to me, if it is that what you want me to review

**Review by MariekeGO — APPROVED** [review]
